### ISSUE TITLE
Cursor not visible at the beginning of a line when editor focus indicator visible

### ISSUE THREAD

**Issue by mbrookes** [jquense/jarle#3]
Steps to reproduce:

1. In the docs, tab into an editor
2. Cursor will be at the start of the first line
3. Note that the cursor isn't visible

It seems it's obscured by the focus indicator, but I haven't dug any deeper. (Incidentally, I noticed that in the docs, only keyboard focus is highlighted, but in my case, clicking in the editor also gives it a blue border?)
